flow/probes/ebpf.go

```diff
@@ -302,9 +333,9 @@ func (p *EBPFProbe) run() {
 					ebpfFlow.lastK = lastK
 					ebpfFlow.last = last
 
-					flowChanOperation <- p.newFlowOperation(ebpfFlow, &kernFlow)
+					flowChanOperation <- p.newFlowOperation(ebpfFlow, &kernFlow, startKTimeNs, start)
```

Comment:
why not passing only us variable calculated above ?

Reply:
my understanding is that us is the time at which the flow started. However, we are not interested in this time here since we want to translate other kernel time stamps (of the flags) so we need the two parameters. Correct?